<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiuyaoPPFrame</class>
 <widget class="QFrame" name="LiuyaoPPFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>edit_name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_name"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Birthday</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCalendarWidget" name="calendar"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="time">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="time">
         <time>
          <hour>12</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="displayFormat">
         <string>hh:mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnNow">
        <property name="text">
         <string>Current Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPaipan">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>PaiPan</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Question</string>
     </property>
     <property name="buddy">
      <cstring>edit_question</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edit_question"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Method</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLineEdit" name="edit_mtd12">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>99999; </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_gua">
        <property name="text">
         <string>Gua code</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radio_coin">
        <property name="text">
         <string>Coins</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="edit_mtd2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>99999999; </string>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_number">
        <property name="text">
         <string>Numbers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="edit_mtd13">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>99999; </string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_mtd11">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>99999; </string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="edit_mtd31">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>9; </string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>First Time</string>
        </property>
        <property name="buddy">
         <cstring>edit_mtd31</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Second Time</string>
        </property>
        <property name="buddy">
         <cstring>edit_mtd32</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Third Time</string>
        </property>
        <property name="buddy">
         <cstring>edit_mtd33</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Forth Time</string>
        </property>
        <property name="buddy">
         <cstring>edit_mtd34</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Fifth Time</string>
        </property>
        <property name="buddy">
         <cstring>edit_mtd35</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Sixth Time</string>
        </property>
        <property name="buddy">
         <cstring>edit_mtd36</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="edit_mtd32">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>9; </string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="edit_mtd33">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>9; </string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="edit_mtd34">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>9; </string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="edit_mtd35">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>9; </string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="edit_mtd36">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="inputMask">
         <string>9; </string>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_name</tabstop>
  <tabstop>edit_question</tabstop>
  <tabstop>radio_number</tabstop>
  <tabstop>edit_mtd11</tabstop>
  <tabstop>edit_mtd12</tabstop>
  <tabstop>edit_mtd13</tabstop>
  <tabstop>radio_gua</tabstop>
  <tabstop>edit_mtd2</tabstop>
  <tabstop>radio_coin</tabstop>
  <tabstop>edit_mtd31</tabstop>
  <tabstop>edit_mtd32</tabstop>
  <tabstop>edit_mtd33</tabstop>
  <tabstop>edit_mtd34</tabstop>
  <tabstop>edit_mtd35</tabstop>
  <tabstop>edit_mtd36</tabstop>
  <tabstop>calendar</tabstop>
  <tabstop>time</tabstop>
  <tabstop>btnNow</tabstop>
  <tabstop>btnPaipan</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnNow</sender>
   <signal>clicked()</signal>
   <receiver>LiuyaoPPFrame</receiver>
   <slot>onBtnNow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_number</sender>
   <signal>toggled(bool)</signal>
   <receiver>LiuyaoPPFrame</receiver>
   <slot>onMethodToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_gua</sender>
   <signal>toggled(bool)</signal>
   <receiver>LiuyaoPPFrame</receiver>
   <slot>onMethodToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_coin</sender>
   <signal>toggled(bool)</signal>
   <receiver>LiuyaoPPFrame</receiver>
   <slot>onMethodToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPaipan</sender>
   <signal>clicked()</signal>
   <receiver>LiuyaoPPFrame</receiver>
   <slot>onBtnPaipan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnPaipan()</slot>
  <slot>onTimeChg(QTime)</slot>
  <slot>onDateChg()</slot>
  <slot>onMethodToggled(bool)</slot>
  <slot>onNameChg(QString)</slot>
  <slot>onBtnNow()</slot>
  <slot>onQuestionChg(QString)</slot>
 </slots>
</ui>
